sharness/lib/test-lib.sh

```diff
@@ -66,6 +68,9 @@ test_cluster_init() {
         echo "error initializing ipfs cluster"
         exit 1
     fi
+    if [[ -n "$custom_config_files" ]]; then
```

Comment:
Ah, nice, I noticed `[` was used throughout and meant to test the difference (but forgot it was an `sh` thing). Thanks, about to push that change.